frontend/server/src/DAO/Tags.php

```diff
@@ -75,4 +75,46 @@ public static function findPublicTagsByPrefix(
 
         return $results;
     }
+
+    /**
+     * @return list<array{name: string}>
+     */
+    public static function getFrequentsByLevel(
+        string $problemLevel
+    ) {
+        $sql = '
+            SELECT
+                t.name
+            FROM
+                Problems_Tags pt
+            INNER JOIN
+                Tags t ON t.tag_id = pt.tag_id
+            WHERE
+                pt.problem_id
+            IN (
+                SELECT
+                    problem_id
+                FROM
+                    Problems_Tags
+                WHERE
+                    tag_id = (
+                        SELECT
+                            tag_id
+                        FROM
+                            Tags
+                        WHERE name = ? )
+            ) AND
+                name LIKE "problemTag%"
+            GROUP BY
+                t.name
+            ORDER BY
+                COUNT(pt.problem_id)
+            ';
+
+        /** @var list<array{name: string}> */
```

Comment:
Acá también cambiaría la anotación:
```suggestion
        /** @var list<array{alias: string}> */
```